src/components/navigation/Navbar.stories.tsx

```diff
@@ -161,6 +176,54 @@ export const CustomChildElement: Story = {
   },
 };
 
+export const LinksInSlot: Story = {
+  args: {
+    rightSlot: [
```

Comment:
Can we get these using just `React.ReactElement<LinkProps>` rather than the list `React.ReactElement<LinkProps>[]` 

And  use the style like below, I think it's "more React-like":
```
<>
      <User
        key="user"
        onLogin={() => {}}
        onLogout={() => {}}
        user={{ name: "Name", fedid: "FedID" }}
        color={"white"}
      />
      <ColourSchemeButton key="colourScheme" />
</>
```

Reply:
This is a comment on some of the other examples too.

Reply:
Updated these.